content/java/concepts/math-methods/terms/to-int-exact/to-int-exact.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: '.toIntExact()'
+Description: 'Returns the integer value of a long type argument. An exception is thrown if the value of argument overflows that of an int.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Methods'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`Math.toIntExact()`** method returns the integer value of a `long` type argument. An [exception](https://www.codecademy.com/resources/docs/java/errors) is thrown if the value of argument overflows that of an `int`.
+
+## Syntax
+
+```pseudo
+Math.toIntExact(long val)
+```
+
+The return type of the `.toIntExact()` method is an `int` type value. The data type of the `val` parameter is `long`.
```

Comment:
~The return type of the `.toIntExact()` method is an `int` type value. The data type of the `val` parameter is `long`.~

Consider this instead:

"In the function `Math.toIntExact(long val)`, the parameter `val` is of type `long`. The function will return an `int`."